lib/IRGen/GenProto.cpp

```diff
@@ -1751,6 +1751,10 @@ class AccessorConformanceInfo : public ConformanceInfo {
 #endif
 
       auto typeWitness = Conformance.getTypeWitness(assocType);
+      if (langOpts.hasFeature(Feature::EmbeddedExistentials) &&
+          SILWT->isSpecialized()) {
+        typeWitness = entry.getAssociatedTypeWitness().Witness;
```

Comment:
This doesn't make sense, why is Conformance.getTypeWitness() returning the wrong thing? Is Conformance is not a SpecializedProtocolConformance? If not, it should be fixed to match up and the conditional removed.